doc/tutorials/using_agl_as_uos.rst

```diff
@@ -0,0 +1,120 @@
+.. _using_agl_as_uos:
+
+Using AGL as the User OS
+########################
+
+This tutorial describes the steps to run Automotive Grade Linux (AGL) 
+as the User OS on ACRN hypervisor and the problems we got at current stage. 
+We hope the steps documented in this article could help us to reproduce the 
+problem much easier and provide some information for further debugging.
+
+.. image:: images/The-overview-of-AGL-as-UOS.png
+   :align: center
+
+Introduction of AGL
+*******************
+
+Automotive Grade Linux (AGL) is an open source project of The Linux Foundation 
+that is building a Linux-based, open software platform for automotive application.
+For more information about AGL, please visit AGL’s official website:
+https://www.automotivelinux.org/.
+
+Steps for using AGL as the UOS
+******************************
+
+#. Hardware preparation
+
+   The regulatory model of NUC we used is `NUC6CAYH 
+   <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc6cayh.html>`_
+   and this kind of NUC has two display ports: 1x HDMI and 1x VGA.
+   So two displays needed for NUC, one for SOS and one for UOS, 
+   and connect these two displays to NUC.
+
+#. Prepare the Service OS (SOS)
+
+   Follow the instructions found in the Getting started guide for Intel NUC 
+   to setup SOS :ref:`getting-started-apl-nuc`.
+
+#. Prepare the User OS (UOS)
```

Comment:
What surprises me in this section is that you do not install the kernel-iot-lts2018 kernel modules in the AGL image/rootfs. Shouldn't we do this?